src/lib/content.d.ts

```diff
@@ -552,6 +555,8 @@ interface Image {
 		height: string;
 		width: string;
 		isMaster?: string;
+		source?: string;
+		caption?: string;
```

Comment:
I added these types after refreshing the fixture content and finding them missing